package.json

```diff
@@ -5,7 +5,10 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build && node server.js",
+    "build": "ng build && npm run webpack",
+    "server": "node dist/server.js",
+    "webpack": "webpack --config webpack.config.js",
```

Comment:
What does this name mean?

try something like build-server and build-server-watch for these commands to make some actual sense

Reply:
And please do not forget to change the name of webpack file as it will be changed in comment below